network/p2p/p2pbuilder/utils.go

```diff
@@ -32,8 +33,8 @@ type limitConfigLogger struct {
 	logger zerolog.Logger
 }
 
-// newLimitConfigLogger creates a new limitConfigLogger.
-func newLimitConfigLogger(logger zerolog.Logger) *limitConfigLogger {
+// NewLimitConfigLogger creates a new limitConfigLogger.
+func NewLimitConfigLogger(logger zerolog.Logger) *limitConfigLogger {
```

Comment:
```suggestion
// newLimitConfigLogger creates a new limitConfigLogger.
func newLimitConfigLogger(logger zerolog.Logger) *limitConfigLogger {
```

Why export this outside the package? It's not used outside the package.

Reply:
https://github.com/onflow/flow-go/pull/4846/commits/0de5c5d81e0fe4f7566e07cd4f383b5d0ec70966